server/public/index.go.html

```diff
@@ -6,17 +6,45 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>{{ __ "offen" }}</title>
   <link rel="stylesheet" type="text/css" href="/normalize.css">
-  <link rel="stylesheet" type="text/css" href="/skeleton.css">
+  <link rel="stylesheet" type="text/css" href="/lit.css">
+  <link rel="stylesheet" type="text/css" href="/fix.css">
   {{ with .rootAccount }}
     <script src="/script.js" data-account-id="{{ . }}"></script>
   {{ end }}
 </head>
 <body>
-  <div class="container">
+  <div class="c">
     <div class="row">
-      <div class="twelve columns">
-        <h1>{{ __ "Welcome to offen" }}</h1>
-        <p>{{ __ "Access your data in the <a href=\"auditorium\">auditorium</a>" }}</p>
+      <h1>{{ __ "Welcome to <strong>offen</strong>" }}</h1>
+    </div>
+    <div class="row"></div>
+    <div class="row">
+      <div class="card">
+          <div class="row">
+            <div class="2 col"></div>
```

Comment:
Empty markup like this is also something we'll need to avoid for accessibility reasons. It looks as if this is how the lit grid works, so I think we will either need to stick to the grid (and use a full width form) or use a different grid?

Reply:
then lets go full width